x-pack/plugins/ml/public/application/explorer/explorer.js

```diff
@@ -332,79 +336,90 @@ export class Explorer extends React.Component {
             />
             <EuiSpacer size="m" />
 
-            {annotationsData.length > 0 && (
-              <>
+            <EuiAccordion
+              id={this.htmlIdGen()}
+              buttonContent={
                 <EuiTitle className="panel-title">
                   <h2>
                     <FormattedMessage
                       id="xpack.ml.explorer.annotationsTitle"
-                      defaultMessage="Annotations"
+                      defaultMessage=" {count, plural, one {Annotation} other {Annotations}} ({count})"
```

Comment:
I think this title can just be `Annotations` even if there is only 1 annotation.

I wonder if the display of the total count e.g. `Annotations (3)` is obvious to the user? Any thoughts @mdefazio, with alternatives being an `EuiBadge` or  something like `Annotations (Total: 3)`. Comparing with visuals used in the data visualizer:

![image](https://user-images.githubusercontent.com/7405507/86268611-5b69d980-bbc0-11ea-8d6f-e993c0c8d0d1.png)

The idea is that it gives the user an idea of whether it is worth expanding this section to see the table.

Reply:
I would go with the Total: X format.  The badge in the data visualizer was a contextual number, correct? That can change based on filters or settings, which I think it more important to highlight and therefore needs the badge.  So since the annotations will only have the total (is this correct?), then I think just `Total: X` works

Reply:
The annotations table is filtered by default, to only show user generated annotations, so this number provides the total of user + system generated ones.

![image](https://user-images.githubusercontent.com/7405507/86271653-2c099b80-bbc5-11ea-8a33-e1f81c08d468.png)

Reply:
Added badge here https://github.com/elastic/kibana/pull/70198/commits/bbdbec654cc6181b24aafed2d13de5f8be81e658

Reply:
I think the badge works well!